packages/rancher-istio/charts/README.md

```diff
@@ -1,26 +1,37 @@
-# Rancher Istio Installers
```

Comment:
Asking again to see if we can remove this. If not its ok.

Reply:
We should split them up so its not duplicated.

Reply:
My most recent commit removes duplicate information - I put the info about installing, uninstall and configuration in the readme and deprecations and know issues in the app-readme